README.md

```diff
@@ -79,9 +88,16 @@ Here's a hands-on exercise to illustrate the power of GitOps:
 
 1. Open the podinfo app and observe its default jade green background.
 2. Dive into the `manifests/podinfo/resources/deployment.yaml`` file to see its configuration. Locate the `PODINFO_UI_COLOR` environment variable and update its value to the golden hue of `#FFD700`.
+
+> Connecting the Git Repo Server with the Argo CD webhook would be user useful here to avoid confusion and automagically apply the updates.
```

Comment:
webhook is actually quite hard because you need to send a certain payload with all the files that changed. I found a previous error now in adjusting the poll frequency. It works now and I adjusted the poll frequency to 5s now. That should be sufficient.

Reply:
5s polling is good for now 🫡